include/caffe/net.hpp

```diff
@@ -32,11 +32,10 @@ class Net {
   void Init(const NetParameter& param);
 
   /**
-   * @brief Run Forward with the input Blob%s already fed separately.
+   * @brief Run Forward and return the result.
    *
-   * You can get the input blobs using input_blobs().
    */
-  const vector<Blob<Dtype>*>& ForwardPrefilled(Dtype* loss = NULL);
```

Comment:
I kept `ForwardPrefilled()` and `Forward(bottom, loss)` for now with deprecation.